web_pivot_computed_measure/tests/res_users_fake.py

```diff
@@ -1,12 +1,7 @@
-# Copyright 2022 Tecnativa - Carlos Roca
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
-
-
```

Comment:
:thinking:

Reply:
sorry sorry, I don't mean to ignore the author :)